### ISSUE TITLE
Add column "indexed" to articles_db

### ISSUE THREAD

**Issue by oliversauter** [WorldBrain/cortex#46]
We need a status for an article, that it has been running through all the steps of being parsed. 

I would suggest using a time stamp, since this would help us later to decide for recrawling of the page.

**Comment by hoshomoh** [comment]
Is this still relevant? @oliversauter

**Comment by oliversauter** [comment]
Resolution from convo in #40:

**Changes in `sources` table:**
- Adding last time crawled
- adding last error message to domain (for now the last one is ok, later on we add a log file to each domain)
- Adding 2 states:
  - `CONTENT_COMPLETE` =content extraction complete and URLs of domain ready for being indexed
  - `INDEXED` = it is indexed in elastic search 

**Changes in `all_urls` table**
Adding 3 states:
-  change `PROCESSED` to `PARSED` (processed content extractor)
-  `FAILED`  
-  `INDEXED` (indexed by elastic search)

**Comment by vpolisky** [comment]
In the Source model, there is the following transition:
```
    @transition(
        field=state,
        source=SourceStates.CRAWLED.value,
        target=SourceStates.FAILED.value,
        custom=dict(admin=False)
    )
```
Why do we need this one?  As I understand from [this discussion](https://github.com/WorldBrain/cortex/issues/40), a domain becomes FAILED if an error occurs while crawling. So, there should be a transition like READY -> FAILED?

**Comment by hoshomoh** [comment]
In real sense a domain should be able to transition from any state to `FAILED`. So you might want to change the source to `source='*'`

**Comment by oliversauter** [comment]
is this done?

**Comment by hoshomoh** [comment]
It's merged already #57
